### PR TITLE
Added devrantio app (fixed old pull request)

### PR DESCRIPTION
**Pull request by mainrs**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

@vitorgalvao

### CODE REVIEW

File: Casks/devrantio.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'devrantio' do
+    version '1.0.2'
+    sha256 '440a39b8e16cde4e8fc55eb387526bc74c5176dcad1c80820a1582b350ac870e'
+    
+    # github.com was verified as official when first introduced to the cask
+    url "https://github.com/Meadowcottage/devRant/releases/download/#{version}/devRant.io-darwin-x64.zip"
+    appcast 'https://github.com/Meadowcottage/devRant/releases.atom',
+    checkpoint: '1292b2981cd6f3e90fc0a5b0defaffc373113dda3455c0b8b02e6aceaeaff0a1'
```
Comment (vitorgalvao) [inline Casks/devrantio.rb]: This line needs to be indented.

File: Casks/devrantio.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    name 'devrantio'
+    homepage 'https://devrant.io/'
+    license :gpl
+    
+    app 'devRant.io-darwin-x64/devrant.io.app'
+end
```
Comment (vitorgalvao) [inline Casks/devrantio.rb]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

File: Casks/devrantio.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+cask 'devrantio' do
+    version '1.0.2'
+    sha256 '440a39b8e16cde4e8fc55eb387526bc74c5176dcad1c80820a1582b350ac870e'
+    
+    # github.com was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/devrantio.rb]: Comment needs to be more specific (`github.com/Meadowcottage/devRant`).

File: Casks/devrantio.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    url "https://github.com/Meadowcottage/devRant/releases/download/#{version}/devRant.io-darwin-x64.zip"
```
Comment (vitorgalvao) [inline Casks/devrantio.rb]: How did you find this? I see nothing on the homepage that points here.

File: Casks/devrantio.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    appcast 'https://github.com/Meadowcottage/devRant/releases.atom',
+    checkpoint: '1292b2981cd6f3e90fc0a5b0defaffc373113dda3455c0b8b02e6aceaeaff0a1'
+    name 'devrantio'
+    homepage 'https://devrant.io/'
```
Comment (vitorgalvao) [inline Casks/devrantio.rb]: This app you’re linking to is very much unofficial, so this is not the homepage. Use the github homepage instead and remove the `was verified as official when first introduced to the cask` comment.

File: Casks/devrantio.rb [hunk 6]
```diff
@@ -0,0 +1,14 @@
+cask 'devrantio' do
```
Comment (vitorgalvao) [inline Casks/devrantio.rb]: Keep this name. It is correct.

File: Casks/devrantio.rb (hunk 3 above)
Comment (vitorgalvao) [inline Casks/devrantio.rb]: [Remove comment](https://github.com/caskroom/homebrew-cask/pull/22778#discussion_r70713778).

**Comment by SirWindfield** [comment]
@vitorgalvao hopefully I've fixed all problems. If not, just mark them and add a comment :)

**Comment by vitorgalvao** [comment]
Looks good, thank you. Just waiting for Travis.

**Pull request merged**